### PR TITLE
Preprocess first batch of data

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Preprocess first batch of data

In this PR, the first batch of data was preprocessed to extract single-cell profiles, perform single-cell QC, and format with pycytominer. Some figures are generated from the single-cell QC.

### CODE REVIEW

File: 3.preprocessing_features/qc_figures/localhost240927060001_nuclei_outliers.png
Comment (jaceybronte) [inline 3.preprocessing_features/qc_figures/localhost240927060001_nuclei_outliers.png:1]: This is cool!

File: 3.preprocessing_features/scripts/0.convert_cytotable.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    ].rename(
+        columns=lambda col: (
+            "Metadata_" + col
+            if col
+            in [
+                "Nuclei_Location_Center_X",
+                "Nuclei_Location_Center_Y",
+                "Cells_Location_Center_X",
+                "Cells_Location_Center_Y",
+                "Image_Count_Cells",
```
Comment (jaceybronte) [inline 3.preprocessing_features/scripts/0.convert_cytotable.py:126]: This seems repetitive - could it be consolidated and have the list of columns defined just once and then used from there?

File: 3.preprocessing_features/scripts/1.sc_quality_control.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+# `We determine our own threshold using the first batch of data (4 plates from layout one) that we will use for the rest of the plates.`
+# 
+# ### Assessing poor cell segmentation
+# 
+# Also due to high confluence, images with large, intense clusters of cells leads to errors in the segmentation algorithm that causes cells around the cluster to segmented incorrectly. 
+# When this happens, a cell is segmented around the same segmentation as the nucleus, giving it the same area which is very small for a normal cardiac fibroblast cell. To detect poorly segmented cells, we use:
+# 
+# - **Cells area:** The cells Area metric quantifies the number of pixels in a cell segmentation.
+# 
+# `We update the threshold for this dataset from the original thresholds since we determined that there were too many good cells being removed. We update the threshold to be more loose.`
```
Comment (jaceybronte) [inline 3.preprocessing_features/scripts/1.sc_quality_control.py:32]: Love the documentation!

File: 3.preprocessing_features/scripts/0.convert_cytotable.py (hunk 1 above)
Comment (jaceybronte) [inline 3.preprocessing_features/scripts/0.convert_cytotable.py:126]: This seems repetitive - could it be consolidated and have the list of columns defined just once and then used from there?

File: 3.preprocessing_features/scripts/1.sc_quality_control.py (hunk 2 above)
Comment (jaceybronte) [inline 3.preprocessing_features/scripts/1.sc_quality_control.py:32]: Love the documentation!

**Review by jaceybronte — APPROVED** [review]
LGTM! Very fun to review, I think I only had one or two comments

File: 3.preprocessing_features/qc_figures/localhost240927060001_nuclei_outliers.png
Comment (jenna-tomkinson) [reply 3.preprocessing_features/qc_figures/localhost240927060001_nuclei_outliers.png:1]: Thank you! My next goal is to move this plot into R and make this same scatterplot + KDE plots on the edges to show how there are a ton more green cells than what it looks like (it is a bit misleading given all of the spread-out "bad" cells).

File: 3.preprocessing_features/scripts/0.convert_cytotable.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 3.preprocessing_features/scripts/0.convert_cytotable.py:126]: Hmm, great point! I will do just that, thanks!

File: 3.preprocessing_features/scripts/1.sc_quality_control.py (hunk 2 above)
Comment (jenna-tomkinson) [reply 3.preprocessing_features/scripts/1.sc_quality_control.py:32]: Thank you!

File: 3.preprocessing_features/scripts/0.convert_cytotable.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 3.preprocessing_features/scripts/0.convert_cytotable.py:126]: Updated!

File: 3.preprocessing_features/scripts/1.sc_quality_control.py (hunk 2 above)
Comment (jenna-tomkinson) [reply 3.preprocessing_features/scripts/1.sc_quality_control.py:32]: Thank you! I hope it made sense all the quality control stuff lol. I need to include the image data at some point in here.

**Comment by jenna-tomkinson** [comment]
@jaceybronte Thank you lots for reviewing! I will be merging now!

**Pull request merged**